### PR TITLE
Add/home page/routing

### PR DESCRIPTION
**Pull request by frank12421**

### CODE REVIEW

File: components/CreateCard.js [hunk 1]
```diff
@@ -1,31 +1,67 @@
...
+  align-items: center;
+  gap: 20px;
+`;
+
+const ListElementStyled = styled.p`
+  margin: 0;
+`;
+
 export default function CreateCard({ data, type }) {
   if (type === "plant") {
```
Comment (actyralf) [inline components/CreateCard.js:41]: You are using `type` to decide wether to render a place or a plant. These two objects do not have a lot in common, you only want to apply the same styling to the container. 
This makes your card component very unflexible - if you want to use it with a 3rd type of content you would have to add another `if` block and so on. 
A better approach  would be to create a container component (e.g. `Card`) that simply renders children. When you instantiate this component (e.g. in `placelist.js` or `plantlist.js`), you can then use the appropriate content component as child.

**Review by actyralf — CHANGES_REQUESTED** [review]
Well done!
My main concern though is your `CreateCard` component that uses `if` blocks to render completely different content based on `type`, which makes it quite unflexible.
Additionally you might want to consider some naming convention proposals.

**Comment by frank12421** [comment]
Hello Ralf, thanks for the good tips and your help. I have changed everything and made the code much clearer, I think. 
Please revise again. Thank you very much.

File: components/Card.js
[the remark this answers is not in the document]
Comment (frank12421) [reply components/Card.js:1]: Hi Robin, thank you very much for your comments. I have changed this to Card on the advice of Ralf. Ralf had explained to me that styled components are actually normal components and can also be used in this way. https://github.com/frank12421/capstone-project/pull/15#discussion_r1189615820

**Pull request merged**